bot.js

```diff
@@ -55,7 +55,48 @@ bot.on("message", async message => {
     let args = messageArray.slice(1);
     let prefix = serverSettings.prefix;
     if (!command.startsWith(prefix)) return;
-
+    
+    /** Command: Translate
+     * Description: Translates text into result language.
+     * Notes: Utilizes Google's translate api. Requires https://www.npmjs.com/package/google-translate-api
+     * Usage: {prefix}translate ko en 사란 해
+     *        {prefix}translate korean english 사란 해
+     */ 
+   
+    //https://www.npmjs.com/package/google-translate-api
+    //const translate = require('google-translate-api'); <---requires this******
+    //^^^^^^^^^
+    
+    if (command === `${prefix}translate`) {
+        //Ex. Message: !translate ko en 사란 해
+        let args = message.content.split(" ").slice(1); //args = ["ko", "en", "사란 해"]
+        let langFrom = args[0].toLowerCase(); //"ko"
+        if (langFrom === "to") langFrom = "auto"; //handles if Message: !translate to en 사란 해
+        
+        let langTo = args[1].toLowerCase(); //"en"
+        if (langFrom === "chinese" || langTo === "chinese") { //2 different chinese, must specify
+            message.channel.send("Please specify which Chinese to translate and try again i.e. chinese-simplified/chinese-traditional or zh-cn/zh-tw");
+            return;
+        }
+        //theres a better way ik
```

Comment:
what do you mean?

Reply:
Nah tbh not really actually, a switch would prob be longer i just wrote that cuz it was arduous